presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -44,17 +43,23 @@
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public class TestElasticsearchIntegrationSmokeTest
+public abstract class TestElasticsearchIntegrationSmokeTest
```

Comment:
Let's call this class `BaseElasticsearchSmokeTest`

Reply:
changed